setup.py

```diff
@@ -122,6 +122,7 @@ def run(self):
         'flask-cache>=0.13.1, <0.14',
         'flask-login==0.2.11',
         'future>=0.15.0, <0.16',
+        'funcsigs>=0.4, <1'
```

Comment:
`setup.py` is my kryptonite... Fixed in 6581858703211bad60793f4b278871edb842fd46